integration/configs/integrationTest/echo_reduction_test.config.json

```diff
@@ -1,39 +1,36 @@
 {
-    "`": 1,
-    "name": "echo_reduction_test.config.json %ts",
-    "grids": [],
-    "tests": [
-      {
-        "name": "Echo Reduction Enabled Test %ts",
-        "description": "Echo Reduction Enabled Test",
-        "tupleSize": 1,
-        "noOfThreads": 1,
-        "testImpl": "EchoReductionEnabledTest.js",
-        "payload": {
-          "url": "http://localhost:8080/?voiceFocusRevisionID=ptCWCoMSFmUNqs6JXcVamv",
-          "port": 30002,
-          "retry": 5
-        }
-      },
-      {
-        "name": "Echo Reduction Disabled Test %ts",
-        "description": "Echo Reduction Disabled Test",
-        "tupleSize": 1,
-        "noOfThreads": 1,
-        "testImpl": "EchoReductionDisabledTest.js",
-        "payload": {
-          "url": "http://localhost:8080/",
-          "port": 30002,
-          "retry": 5
-        }
+  "`": 1,
+  "name": "echo_reduction_test.config.json %ts",
+  "grids": [],
+  "tests": [
+    {
+      "name": "Echo Reduction Enabled Test %ts",
+      "description": "Echo Reduction Enabled Test",
+      "tupleSize": 1,
+      "noOfThreads": 1,
+      "testImpl": "EchoReductionEnabledTest.js",
+      "payload": {
+        "url": "http://localhost:8080/?voiceFocusRevisionID=ptCWCoMSFmUNqs6JXcVamv",
+        "retry": 5
```

Comment:
Should the retry be 2 like everything else?

Reply:
@nainkunal933 Is it okay to reduce this `retry` from 5 to 2?

Reply:
@nainkunal933 I will change the `retry` to 2 for now. Let me know if you have any concern so I can revert it.